src/workerd/jsg/async-context.h

```diff
@@ -83,7 +83,7 @@ class AsyncContextFrame final: public Wrappable {
     bool isDead() const { return dead; }
     inline uint hashCode() const { return hash; }
     inline bool operator==(const StorageKey& other) const {
-      return hash == other.hash;
+      return this == &other;
```

Comment:
Hmm... I was thinking maybe `StorageKey` should be `KJ_DISALLOW_COPY_AND_MOVE`, but I guess that's implicit in its inheritance from `Refcounted`?

Reply:
No harm in being more explicit about it